.github/workflows/ci.yml

```diff
@@ -175,7 +175,7 @@ jobs:
         aws-region: ${{ env.AWS_DEFAULT_REGION }}
         role-duration-seconds: 14400 # these tests run slow and easily reach default cred expiry, hence change expiry to 4hrs
     - name: Install qemu/docker
-      run: docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+      run: docker run --privileged --rm tonistiigi/binfmt --install all
```

Comment:
Should we have qemu for linux-arm CI as well?

Reply:
good question. i have no idea how its currently running at all. let me check

Reply:
linux arm is cross-compiling, we don't need qemu for that

Reply:
yeah, its running dockcross through builder there, so we are fine